stress: failed test in cockroach/storage/storage.test: TestAcceptsUnsplitRanges

Binary: cockroach/static-tests.tar.gz sha: https://github.com/cockroachdb/cockroach/commits/bebc77abe79ef5904f8fa249f0b2c359c8acfbb6

Stress build found a failed test:

```
=== RUN   TestAcceptsUnsplitRanges
I160925 05:22:55.317354 1186 gossip/gossip.go:234  initial resolvers: []
W160925 05:22:55.317379 1186 gossip/gossip.go:1045  no resolvers found; use --join to specify a connected node
I160925 05:22:55.317441 1186 gossip/gossip.go:268  NodeID set to 1
I160925 05:22:55.317511 1186 storage/engine/rocksdb.go:356  opening in memory rocksdb instance
I160925 05:22:55.318880 1186 gossip/gossip.go:268  NodeID set to 1
I160925 05:22:55.318991 1186 gossip/gossip.go:268  NodeID set to 1
I160925 05:22:55.322547 1194 storage/replica_trigger.go:254  [s1,r1:/M{in-ax}] new range lease replica {1 1 1} 1970-01-01 00:00:00 +0000 UTC 900ms following replica {0 0 0} 1970-01-01 00:00:00 +0000 UTC 0s [physicalTime=1970-01-01 00:00:00 +0000 UTC]
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xad0f6c]

goroutine 1171 [running]:
panic(0x1678500, 0xc4200100e0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/cockroachdb/cockroach/util/stop.(*Stopper).Recover(0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:177 +0x6e
panic(0x1678500, 0xc4200100e0)
	/usr/local/go/src/runtime/panic.go:458 +0x243
github.com/coreos/etcd/raft.(*raft).appendEntry(0xc4204c80f0, 0xc42094acd0, 0x1, 0x1)
	/go/src/github.com/coreos/etcd/raft/raft.go:495 +0xdc
github.com/coreos/etcd/raft.(*raft).becomeLeader(0xc4204c80f0)
	/go/src/github.com/coreos/etcd/raft/raft.go:581 +0x1ce
github.com/coreos/etcd/raft.(*raft).campaign(0xc4204c80f0, 0x17cbabe, 0x10)
	/go/src/github.com/coreos/etcd/raft/raft.go:588 +0x709
github.com/coreos/etcd/raft.(*raft).Step(0xc4204c80f0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/coreos/etcd/raft/raft.go:636 +0x243
github.com/coreos/etcd/raft.(*RawNode).Campaign(0xc4202d4980, 0xc4202d4900, 0x0)
	/go/src/github.com/coreos/etcd/raft/rawnode.go:142 +0x71
github.com/cockroachdb/cockroach/storage.(*Replica).withRaftGroupLocked(0xc4209c6300, 0x0, 0xc4204a57f0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:444 +0x31f
github.com/cockroachdb/cockroach/storage.defaultProposeRaftCommandLocked(0xc4209c6300, 0xc4204ca000, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:1726 +0x1ed
github.com/cockroachdb/cockroach/storage.(*Replica).proposePendingCmdLocked(0xc4209c6300, 0xc4204ca000, 0xc4201870e0, 0xc4204b4148)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:1665 +0x77
github.com/cockroachdb/cockroach/storage.(*Replica).proposeRaftCommand(0xc4209c6300, 0x7f53751a4800, 0xc4201870e0, 0x0, 0x16, 0x0, 0x0, 0x3, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:1644 +0x1f0
github.com/cockroachdb/cockroach/storage.(*Replica).addWriteCmd(0xc4209c6300, 0x7f53751a4800, 0xc4201870e0, 0x0, 0x16, 0x0, 0x0, 0x3, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:1529 +0x17a
github.com/cockroachdb/cockroach/storage.(*Replica).Send(0xc4209c6300, 0x7f53751a4800, 0xc4201870e0, 0x0, 0x16, 0x0, 0x0, 0x3, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:1061 +0x277
github.com/cockroachdb/cockroach/storage.(*pendingLeaseRequest).InitOrJoinRequest.func1(0x7f53751a4800, 0xc4203f8390)
	/go/src/github.com/cockroachdb/cockroach/storage/replica_range_lease.go:127 +0x174
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunAsyncTask.func1(0xc4202f43f0, 0x1e98fbc, 0x1e, 0x91, 0x1950a78, 0xc420186f00, 0x7f53751a4800, 0xc4203f8390)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:236 +0xc6
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunAsyncTask
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:237 +0x227

goroutine 1 [chan receive]:
testing.(*T).Run(0xc4202fa240, 0x17d4902, 0x18, 0x194f940, 0xc4202afa01)
	/usr/local/go/src/testing/testing.go:647 +0x316
testing.RunTests.func1(0xc4202fa240)
	/usr/local/go/src/testing/testing.go:793 +0x6d
testing.tRunner(0xc4202fa240, 0xc42026dbc0)
	/usr/local/go/src/testing/testing.go:610 +0x81
testing.RunTests(0x19518a8, 0x21aa680, 0x14b, 0x14b, 0xc42013cce0)
	/usr/local/go/src/testing/testing.go:799 +0x2f5
testing.(*M).Run(0xc42026dee8, 0xc4202afd30)
	/usr/local/go/src/testing/testing.go:743 +0x85
github.com/cockroachdb/cockroach/storage_test.TestMain(0xc42026dee8)
	/go/src/github.com/cockroachdb/cockroach/storage/main_test.go:57 +0x1ff
main.main()
	github.com/cockroachdb/cockroach/storage/_test/_testmain.go:724 +0xc6

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2086 +0x1

goroutine 34 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x24f1d20)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1032 +0x77
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:613 +0x95

goroutine 1186 [runnable]:
time.Sleep(0x1)
	/usr/local/go/src/runtime/time.go:59 +0xe1
github.com/cockroachdb/cockroach/util.RetryForDuration(0xa7a358200, 0xc420415e28, 0x1c, 0xc420415b68)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:136 +0x108
github.com/cockroachdb/cockroach/util.SucceedsSoonDepth(0x1, 0x21c4ec0, 0xc4201180c0, 0xc420415e28)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:115 +0x41
github.com/cockroachdb/cockroach/util.SucceedsSoon(0x21c4ec0, 0xc4201180c0, 0xc420415e28)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:109 +0x48
github.com/cockroachdb/cockroach/storage.TestAcceptsUnsplitRanges(0xc4201180c0)
	/go/src/github.com/cockroachdb/cockroach/storage/queue_test.go:521 +0x8e5
testing.tRunner(0xc4201180c0, 0x194f940)
	/usr/local/go/src/testing/testing.go:610 +0x81
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:646 +0x2ec

goroutine 1216 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146c80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1213 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146c00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1192 [runnable]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146740)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1223 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146da0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1191 [select]:
github.com/cockroachdb/cockroach/storage.(*bookie).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/reservation.go:274 +0x217
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4201edee0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1203 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146aa0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1195 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146960)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1217 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146ca0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1197 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4201469a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1194 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146940)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1229 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func2()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1052 +0x314
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4209e4ca0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1201 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146a60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1205 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146ae0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1193 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4201467a0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1218 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146cc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1221 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146d20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1219 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146ce0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1215 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146c40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1228 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:1033 +0x314
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4209e4c90)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1238 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146fc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1187 [chan receive]:
github.com/cockroachdb/cockroach/rpc.NewContext.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:138 +0x76
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420247960)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1196 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146980)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1212 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146be0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1209 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146b80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1210 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146ba0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1200 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146a40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1202 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146a80)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1199 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146a20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1188 [select]:
github.com/cockroachdb/cockroach/storage.(*StorePool).start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store_pool.go:340 +0x3da
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4201ecd20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1198 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4201469e0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1211 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146bc0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1189 [chan receive]:
github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open.func1(0xc42017c9c0)
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:390 +0x44
created by github.com/cockroachdb/cockroach/storage/engine.(*RocksDB).Open
	/go/src/github.com/cockroachdb/cockroach/storage/engine/rocksdb.go:391 +0x6eb

goroutine 1206 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146b00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1207 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146b20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1208 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146b40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1214 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146c20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1204 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146ac0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1227 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:958 +0x165
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146e20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1224 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2866 +0x334
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4209e4c30)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1225 [select]:
github.com/cockroachdb/cockroach/storage.(*Store).raftSnapshotStatusLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:2896 +0x211
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4209e4c40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1233 [select]:
github.com/cockroachdb/cockroach/storage.(*Replica).redirectOnOrAcquireLease(0xc4209c6300, 0x7f53726ae170, 0xc4200143c0, 0x12)
	/go/src/github.com/cockroachdb/cockroach/storage/replica.go:812 +0x502
github.com/cockroachdb/cockroach/storage.(*baseQueue).processReplica(0xc420180b60, 0xc4209c6300, 0xc420140680, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:514 +0x5fb
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1.2()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:460 +0x58
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunTask(0xc4202f43f0, 0xc4204c5e78, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:216 +0x100
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:464 +0x296
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146ee0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1220 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146d00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1222 [select]:
github.com/cockroachdb/cockroach/storage.(*raftScheduler).worker(0xc42019af50, 0xc4202f43f0)
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:168 +0x3e7
github.com/cockroachdb/cockroach/storage.(*raftScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scheduler.go:160 +0x33
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146d60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1232 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146ec0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1234 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146f00)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1235 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146f20)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1236 [select]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146f40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1237 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitEnabled(0xc420184300, 0xc4202f43f0, 0xc420184310)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:297 +0x1db
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:250 +0xf2
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146f60)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1239 [select]:
github.com/cockroachdb/cockroach/storage.(*replicaScanner).waitAndProcess(0xc420184380, 0xecf79562f, 0xc4130e38b5, 0x24f0c20, 0xc420140680, 0xc4202f43f0, 0xc4201fc300, 0xc420470ec8)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:201 +0x237
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1.1(0xc4201fc300, 0xc420216570)
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:259 +0x7c
github.com/cockroachdb/cockroach/storage.(*storeRangeSet).Visit(0xc4202174a0, 0xc420141180)
	/go/src/github.com/cockroachdb/cockroach/storage/store.go:252 +0x21b
github.com/cockroachdb/cockroach/storage.(*replicaScanner).scanLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/scanner.go:261 +0x233
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc420146fe0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66

goroutine 1250 [runnable]:
github.com/cockroachdb/cockroach/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/storage/queue.go:439 +0x2cc
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc4202f43f0, 0xc4201756c0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:188 +0x7d
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:189 +0x66


ERROR: exit status 2
```

Run Details:

```
0 runs so far, 0 failures, over 5s
0 runs so far, 0 failures, over 10s
0 runs so far, 0 failures, over 15s
0 runs so far, 0 failures, over 20s
16 runs so far, 0 failures, over 25s
16 runs so far, 0 failures, over 30s
16 runs so far, 0 failures, over 35s
16 runs so far, 0 failures, over 40s
32 runs so far, 0 failures, over 45s
32 runs so far, 0 failures, over 50s
32 runs so far, 0 failures, over 55s
32 runs so far, 0 failures, over 1m0s
43 runs completed, 1 failures, over 1m4s
FAIL
```
Please assign, take a look and update the issue accordingly.

--- a/storage/client_test.go
+++ b/storage/client_test.go
@@ -533,6 +533,7 @@ func (m *multiTestContext) makeContext(i int) storage.StoreContext {
 	ctx.Gossip = m.gossips[i]
 	ctx.StorePool = m.storePools[i]
 	ctx.TestingKnobs.DisableSplitQueue = true
+	ctx.TestingKnobs.ReplicateQueueAcceptsUnsplit = true
 	return ctx
 }
 

--- a/storage/queue.go
+++ b/storage/queue.go
@@ -331,10 +331,10 @@ func (bq *baseQueue) requiresSplit(cfg config.SystemConfig, repl *Replica) bool 
 	if bq.acceptsUnsplitRanges {
 		return false
 	}
-	// If there's no store (as is the case in some narrow unit tests), or if
-	// the store's split queue is disabled, the "required" split will never
-	// come. In that case, pretend we don't require the split.
-	if store := repl.store; store == nil || store.splitQueue.Disabled() {
+	// If there's no store (as is the case in some narrow unit tests),
+	// the "required" split will never come. In that case, pretend we
+	// don't require the split.
+	if store := repl.store; store == nil {
 		return false
 	}
 	desc := repl.Desc()

--- a/storage/replicate_queue.go
+++ b/storage/replicate_queue.go
@@ -58,7 +58,7 @@ func newReplicateQueue(store *Store, g *gossip.Gossip, allocator Allocator, cloc
 	rq.baseQueue = makeBaseQueue("replicate", rq, store, g, queueConfig{
 		maxSize:              replicateQueueMaxSize,
 		needsLease:           true,
-		acceptsUnsplitRanges: false,
+		acceptsUnsplitRanges: store.TestingKnobs().ReplicateQueueAcceptsUnsplit,
 		successes:            store.metrics.ReplicateQueueSuccesses,
 		failures:             store.metrics.ReplicateQueueFailures,
 		pending:              store.metrics.ReplicateQueuePending,

--- a/storage/store.go
+++ b/storage/store.go
@@ -565,6 +565,10 @@ type StoreTestingKnobs struct {
 	DisableRefreshReasonTicks bool
 	// DisableProcessRaft disables the process raft loop.
 	DisableProcessRaft bool
+	// ReplicateQueueAcceptsUnsplit allows the replication queue to
+	// process ranges that need to be split, for use in tests that use
+	// the replication queue but disable the split queue.
+	ReplicateQueueAcceptsUnsplit bool
 }
 
 var _ base.ModuleTestingKnobs = &StoreTestingKnobs{}

--- a/storage/store_test.go
+++ b/storage/store_test.go
@@ -172,6 +172,9 @@ func createTestStore(t testing.TB) (*Store, *hlc.ManualClock, *stop.Stopper) {
 	// so just disable the scanner for all tests that use this function
 	// instead of figuring out exactly which tests need it.
 	ctx.TestingKnobs.DisableScanner = true
+	// The scanner affects background operations; we must also disable
+	// the split queue separately to cover event-driven splits.
+	ctx.TestingKnobs.DisableSplitQueue = true
 	return createTestStoreWithContext(t, &ctx)
 }
 

